src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20210908110527_InitialGlossaryEntries.sql

```diff
@@ -0,0 +1,104 @@
+﻿-- Create initial glossary entries
+
+INSERT INTO dbo.GlossaryEntries
+(Id, Created, CreatedById, Title, Slug, Body)
+VALUES
+-- A
+(NEWID(), GETUTCDATE(), 'b99e8358-9a5e-4a3a-9288-6f94c7e1e3dd', 'Absence', 'absence', N'<p>When a pupil misses (or is absent from) at least 1 possible school session.</p><p>Counted in sessions, where 1 session is equivalent to half-a-day.</p><p>There are 4 types of absence:</p><ul><li><a href="/glossary#authorised-absence">authorised absence</a></li><li><a href="/glossary#overall-absence">overall absence</a></li><li><a href="/glossary#persistent-absence">persistent absence</a></li><li><a href="/glossary#unauthorised-absence">unauthorised absence</a></li></ul>'),
```

Comment:
Just letting you know you could have done something like 

```DECLARE @bau1UserId uniqueidentifier = 'b99e8358-9a5e-4a3a-9288-6f94c7e1e3dd';```

or

```DECLARE @bau1UserId uniqueidentifier = (SELECT Id FROM Users WHERE Email = 'ees-bau1@education.gov.uk');```

and used it like

```INSERT INTO dbo.GlossaryEntries (Id, Created, CreatedById, Title, Slug, Body) VALUES
(NEWID(), GETUTCDATE(), @bau1UserId, 'Absence' ...
```

to make it a bit clearer what it is, and in the second example make sure its the correct id.